### PR TITLE
#2559. Make more augmented expression tests stronger

### PR DESCRIPTION
**Pull request by sgrekhov**

### CODE REVIEW

File: LanguageFeatures/Augmentation-libraries/augmented_expression_A03_t13.dart [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+/// - Augmenting fields: Within an augmenting field, augmented can only be used
+///   in an initializer expression, and refers to the original field's
+///   initializer expression, which is immediately evaluated.
+///
+/// If augmented refers to a variable declaration (as defined by a declaration
+/// and a number of prior augmentations) with no initializer expression, and the
+/// variable's type is nullable, augmented evaluates to null. If the variable's
+/// type is not nullable, then it's a compile-time error.
+///
+/// @description Checks that it is a compile-time error ot declare a local
```
Comment (eernstg) [inline LanguageFeatures/Augmentation-libraries/augmented_expression_A03_t13.dart:18]:
```suggestion
/// @description Checks that it is a compile-time error to declare a local
```

File: LanguageFeatures/Augmentation-libraries/augmented_expression_A03_t13_lib.dart [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+/// - Augmenting fields: Within an augmenting field, augmented can only be used
+///   in an initializer expression, and refers to the original field's
+///   initializer expression, which is immediately evaluated.
+///
+/// If augmented refers to a variable declaration (as defined by a declaration
+/// and a number of prior augmentations) with no initializer expression, and the
+/// variable's type is nullable, augmented evaluates to null. If the variable's
+/// type is not nullable, then it's a compile-time error.
+///
+/// @description Checks that it is a compile-time error ot declare a local
```
Comment (eernstg) [inline LanguageFeatures/Augmentation-libraries/augmented_expression_A03_t13_lib.dart:18]:
```suggestion
/// @description Checks that it is a compile-time error to declare a local
```

**Review by eernstg — COMMENTED** [review]
Looks good, just noted two typos!

File: LanguageFeatures/Augmentation-libraries/augmented_expression_A03_t13_lib.dart (hunk 2 above)
Comment (sgrekhov) [reply LanguageFeatures/Augmentation-libraries/augmented_expression_A03_t13_lib.dart]: Typos fixed

**Review by eernstg — APPROVED** [review]
LGTM

**Pull request merged**